src/analyses/variable-sensitivity/variable_sensitivity_domain.h

```diff
@@ -114,7 +114,6 @@ class variable_sensitivity_domaint:public ai_domain_baset
   bool is_bottom() const override;
   bool is_top() const override;
 
-protected:
```

Comment:
:question: (will check commit history) -> why is this necessary?

Reply:
:no_entry_sign: Commit message does not explain why - could you pull this into own commit and justify why the API is being expanded?

Reply:
We need the ability to evaluate individual expressions to abstract objects from outside the VSD itself to get at the abstract objects... this just isn't something we had to do before, which is why it wasn't public.

We can put this into a commit message once you're happy with the other changes.